superdesk/vocabularies/vocabularies.py

```diff
@@ -73,7 +73,6 @@ class VocabulariesResource(Resource):
         },
         'service': {
             'type': 'dict',
-            'default': {'all': True},
```

Comment:
ok, this will need some ui changes so that new vocabularies will show up in profile management